### PR TITLE
Show empty table row in view #LMR-1657

### PR DESCRIPTION
**Pull request by livthomas**

### CODE REVIEW

File: src/app/core/store/tables/tables.effects.ts [hunk 1]
```diff
@@ -600,43 +602,63 @@ export class TablesEffects {
...
-              actions.push(new TablesAction.MoveCursor({direction: Direction.Down}));
-            }
-          }
+          const {collectionId} = table.config.parts[0];
+          return this.store$.pipe(
+            select(selectCollectionById(collectionId)),
+            take(1),
+            mergeMap(collection => this.collectionPermissionsPipe.transform(collection)),
+            take(1),
+            map(permissions => permissions.write || permissions.writeWithView),
```
Comment (kubedo8) [inline src/app/core/store/tables/tables.effects.ts]: It's enough to test if user has `writeWithView` permission, but you can leave it as it is.

**Comment by kubedo8** [comment]
Is this approach reactive? What about situation when user lost `write` access to collection during opened table?

File: src/app/core/store/tables/tables.effects.ts (hunk 1 above)
Comment (livthomas) [reply src/app/core/store/tables/tables.effects.ts]: I would leave it here so in the future we can remove the logic for adding empty rows elsewhere.

**Comment by livthomas** [comment]
@kubedo8 You're right. I've just added the code to remove the empty row once you lose the permissions.

**Comment by kubedo8** [comment]
Great!

**Pull request merged**